tests/saitests/py3/sai_qos_tests.py

```diff
@@ -104,9 +104,12 @@ def check_leackout_compensation_support(asic, hwsku):
 
 def get_ip_addr():
     val = 1
-    while True:
+    while val < 200:
         val = max(val, (val+1)%250)
         yield "192.0.0.{}".format(val)
+    raise RuntimeError(
+        "The program has run out of IP addresses. "
+        "Please check if the dst port is even used.")
 
 
 def get_tcp_port():
```

Comment:
also need to limit the max loop times for 'get_tcp_port'.
or, have to add checking in the outer loop, as below:

def get_multiple_flows(dp, dst_mac, dst_id, dst_ip, src_vlan, dscp, ecn, ttl, pkt_len, src_details, packets_per_port=1):
... omitted ...
    all_pkts = {}
    for src_tuple in src_details:
        num_of_pkts = 0
        while (num_of_pkts < packets_per_port) **and (pkt_sent_num < 200)**:

Reply:
Can we limit the call of `get_multiple_flows` on Cisco?